guideline-11.md

```diff
@@ -1,16 +1,21 @@
 <h4>11. Plugins must not hijack the admin dashboard.</h4>
 
-While it is permitted for plugins to display messages as to their functionality on the WordPress admin dashboard, they must be done with consideration and care to the usability of the WordPress dashboard. Getting in-between users and their desired actions creates a poor experience for WordPress as a whole.
+Getting in-between users and their desired actions creates a poor experience. It prevents users from being able to complete necessary tasks, and is detrimental to WordPress as a whole.
```

Comment:
Should `in-between` simply be `between`?

Reply:
Updated and rewritten much of that section. Thanks.